pkg/controllers/user/cis/clusterScanHandler.go

```diff
@@ -385,3 +399,42 @@ func (csh *cisScanHandler) deleteApp(appInfo *appInfo) error {
 
 	return nil
 }
+
+func (csh *cisScanHandler) ensureCleanup(cs *v3.ClusterScan) error {
```

Comment:
can owner references be used instead to clean any of this up?

Reply:
The installation tools (helm, sonobuoy) already use owner references. This is a fallback cleaning mechanism in case something bad happens.